docs/app/components/side-navigation/side-navigation.component.html

```diff
@@ -1,5 +1,6 @@
 <div class="side-navigation-container" #container>
     <div class="side-navigation" [style.width.px]="width" [style.height.px]="height" [style.top.px]="top">
+        <ng-content></ng-content>
```

Comment:
If the buttons are now on all pages with side navigation we should move the buttons and logic into the side navigation component